lightning/api_grpc.go

```diff
@@ -855,6 +856,10 @@ func (l *LndGrpcLightningAPI) PayInvoice(ctx context.Context, paymentRequest str
 	req.PaymentRequest = paymentRequest
 	if sats > 0 {
 		req.Amt = sats
+		req.FeeLimitSat = int64(math.Max(100, math.Round(float64(sats)*0.01)))
+	} else {
+		// TODO: parse paymentRequest and get amount
+		req.FeeLimitSat = 3000
```

Comment:
I know you'll be all wtf here, but this is insanely complex to do right because you just have invoice not amount.
If I tried to use https://github.com/lightningnetwork/lnd/blob/master/zpay32/decode.go I'd need chain info, which in LND API I don't have 😢